rest-service/manager_rest/resource_manager.py

```diff
@@ -550,12 +550,6 @@ def _reset_operations(self, execution, execution_token, from_states=None):
                     continue
 
                 update_operation = False
```

Comment:
We can change it back to how this code was before the last commit here (update_operation is always false now)

Reply:
it's not always false (also set by the other thin) but you're right, changed :P